functions-and-operators/tidb-functions.md

```diff
@@ -360,50 +481,96 @@ The output is as follows:
+    - The SQL statement executes an equality query based on all fields of the secondary index, either as a separate `SELECT` or as an internal query generated by `UPDATE`, `DELETE` and so on. The equality query includes two ways: `a = 1` or `a IN (1, 2, ......)`.
 
-Use `user1` to log in and view the resource group bound to the current user:
+- Limitations:
+
+    - Cannot be used in inequality queries.
+    - Cannot be used in queries that contain `OR` mixed with an outmost `AND` operator.
+    - Cannot be used in the `GROUP BY` clause.
+    - Cannot be used in the `ORDER BY` clause.
+    - Cannot be used in the `ON` clause.
+    - Cannot be used in the `WHERE` subquery.
+    - Can be used to scatter unique indexes of only the integer fields.
+    - Might not take effect in composite indexes.
+    - Cannot go through FastPlan process, which affects optimizer performance.
+    - Cannot be used to prepare the execution plan cache.
+
+The following example shows how to use the `TIDB_SHARD()` function.
+
+- Use the `TIDB_SHARD()` function to calculate the SHARD value.
+
+    The following statement shows how to use the `TIDB_SHARD()` function to calculate the SHARD value of `12373743746`:
+
+    ```sql
+    SELECT TIDB_SHARD(12373743746);
+    ```
+
+- The SHARD value is:
+
+    ```sql
+    +-------------------------+
+    | TIDB_SHARD(12373743746) |
+    +-------------------------+
+    |                     184 |
+    +-------------------------+
+    1 row in set (0.00 sec)
+    ```
+
+- Create a shard index using the `TIDB_SHARD()` function:
+
+    ```sql
+    CREATE TABLE test(id INT PRIMARY KEY CLUSTERED, a INT, b INT, UNIQUE KEY uk((tidb_shard(a)), a));
+    ```
+
+## TIDB_VERSION
+
+The `TIDB_VERSION()` function is used to get the version and build details of the TiDB server that you are connected to. You can use this function when reporting issues on GitHub.
 
 ```sql
-SELECT CURRENT_RESOURCE_GROUP();
+SELECT TIDB_VERSION()\G
 ```
 
-```
-+--------------------------+
-| CURRENT_RESOURCE_GROUP() |
-+--------------------------+
-| rg1                      |
-+--------------------------+
+```sql
+*************************** 1. row ***************************
+TIDB_VERSION(): Release Version: v5.1.0-alpha-13-gd5e0ed0aa-dirty
+Edition: Community
+Git Commit Hash: d5e0ed0aaed72d2f2dfe24e9deec31cb6cb5fdf0
+Git Branch: master
+UTC Build Time: 2021-05-24 14:39:20
+GoVersion: go1.13
+Race Enabled: false
+TiKV Min Version: v3.0.0-60965b006877ca7234adaced7890d7b029ed1306
+Check Table Before Drop: false
 1 row in set (0.00 sec)
 ```
 
-Execute `SET RESOURCE GROUP` to set the resource group for the current session to `rg2`, and then view the resource group bound to the current user:
+## VITESS_HASH
+
+The `VITESS_HASH(num)` function is used to hash a number in the same way Vitess does this. This is to aid migration from Vitess to TiDB.
```

Comment:
> | `VITESS_HASH(str)` | The `VITESS_HASH` function returns the hash of a string that is compatible with Vitess' `HASH` function. This is intended to help the data migration from Vitess. |

@dveeden The previous document uses `VITESS_HASH(str)`. Here, it is `VITESS_HASH(num)`. Could you please confirm which one is more accurate? 


I tested this function in v8.1.0 and it seems that TiDB will attempt to truncate a string to a number. Should we describe this behavior here?


```shell
mysql> SELECT VITESS_HASH(123);
+---------------------+
| VITESS_HASH(123)    |
+---------------------+
| 1155070131015363447 |
+---------------------+
1 row in set (0.00 sec)

mysql> SELECT VITESS_HASH("123");
+---------------------+
| VITESS_HASH("123")  |
+---------------------+
| 1155070131015363447 |
+---------------------+
1 row in set (0.01 sec)

mysql> SELECT VITESS_HASH("123a");
+---------------------+
| VITESS_HASH("123a") |
+---------------------+
| 1155070131015363447 |
+---------------------+
1 row in set, 1 warning (0.00 sec)

mysql> show warnings;
+---------+------+-------------------------------------------+
| Level   | Code | Message                                   |
+---------+------+-------------------------------------------+
| Warning | 1292 | Truncated incorrect INTEGER value: '123a' |
+---------+------+-------------------------------------------+
1 row in set (0.00 sec)
```

Reply:
`num` is more accurate.

The conversion, earning etc is generic behavior and not specific to this function.